File: Beginner/CheckAge.py
#Input The Age, if age > 18 print adult , if 10 < age <= 18 print teen, if age < 10 print child
age=int(input("Enter the number:"))
if age>18:
    print("adult")
elif age>10 and age<=18:
    print("teen")
else:
    print("child")
    

File: Beginner/CheckAge.cpp
#include <iostream>
using namespace std;

int main()
{
    int p;
    cout << "Enter age : ";
    cin >> p;
    if ( p > 18)
        cout << "adult"<<endl;
    else if(p >10 && p <= 18)
        cout <<"teen"<<endl;
    else if(p < 10)
        cout <<"child "<<endl;
        // age 10 is not handle
    return 0;
}
